README.md

```diff
@@ -16,8 +16,13 @@ So far we tested Northstar on
 * 64-bit ARM
 * x86_64
```

Comment:
I'd add Android as a target because it's quite different from `aarch64-unknown-linux-gnu`.
Why not list the target triples?